node_modules/oae-core/editprofile/css/editprofile.css

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#editprofile-plea {
+    font-size: 14px;
+    font-style: normal;
+    margin-bottom: 15px;
+}
+
+#editprofile-email-verification .alert {
```

Comment:
In going through a 6,500 line PR at often ungodly times of the day or night, I fall back to a mode where I try to make the comments as efficiently as possible and sometimes don't go back and reconsider every word to account for the sensitivities of the programmer. I would hope that when something is inconsiderately worded like "There's no need for this widget to use flexbox.", a simple reply that explains why it is chosen would be sufficient and no insult would be inferred unnecessarily. Having said that, I'll try to be more considerate in the review wording going forward.

Going back on topic, the default approach here would be a margin-right on the left column and floating the button to the right. However, I missed the fact that the button is supposed to be vertically aligned, which rules out that approach. An alternative approach could be the use of `display:table`, `display:table-cell` and `vertical-align: middle`, which would have the desired effect and wouldn't require IE9 specific overrides. Having said that, this approach might be more difficult to make the responsive breakpoint work. If that turns out to be the case, flexbox would be a fine fallback solution.